server/auth.ts

```diff
@@ -15,10 +36,17 @@ export const authOptions: NextAuthOptions = {
       clientId: process.env.GITHUB_ID || "",
       clientSecret: process.env.GITHUB_SECRET || "",
     }),
+    EmailProvider({
+      server: {},
+      sendVerificationRequest: sendPasswordLessEmail,
+      from: process.env.EMAIL_FROM,
```

Comment:
Is this a new variable? Could we use `ADMIN_EMAIL`. I think this is the very last thing (I kinda promise 😂).

Reply:
@NiallJoeMaher sorted in #[19379a8](https://github.com/codu-code/codu/pull/785/commits/19379a8b771118eb454d887c690df6e7cd377d72)